.github/workflows/build-test.yaml

```diff
@@ -654,6 +654,7 @@ jobs:
           kubectl -n default create secret generic multi-namespace-yeti-registry --type=kubernetes.io/dockerconfigjson --from-file=.dockerconfigjson=./replicated-registry-auth.json
 
           kustomize build $PWD/$APP_SLUG/overlays/midstream | kubectl apply -f -
+          kustomize build $PWD/$APP_SLUG/overlays/midstream/charts/redis | kubectl apply -f -
```

Comment:
we now write additional "entrypoints" for kustomize for each useHelmInstall chart